### PR TITLE
 Fix: Mobile responsiveness for Query Output header 

### PR DESCRIPTION
**Pull request by 227001**
I apologize for changing the size of the 'Query took' box. That's why I've created this PR to revert the 'Query took' box to its original size.
截图57
截图22

### CODE REVIEW

File: src/components/Output/OutputDisplay.js [hunk 1]
```diff
@@ -38,43 +38,72 @@ const OutputDisplay = ({ submittedQuery, loading, setLoading }) => {
...
+                    />
+                    <Heading as="h1" fontSize="xl" mt={4}>
+                        Nothing to show at the moment
+                    </Heading>
+                    <Text>Run a query first to see resulting tablet</Text>
+                </VStack>
+            )}
+        </Box>
+    );
 };
```
Comment (janvi01) [inline src/components/Output/OutputDisplay.js:108]:
```suggestion
    return (
    <>
      {results.length > 0 ? (
        <>
          <Stack
            direction={["column", "row"]}
            w={"100%"}
            px={4}
            justifyContent={"space-between"}
          >
            <Heading textAlign="center" fontSize={"3xl"}>
              Query Output
            </Heading>
            <Spacer />
            <HStack justify={"center"}>
              <Button colorScheme="blue" cursor="initial" size={"xs"}>
                Query took: {queryTime}
              </Button>
              <CsvDownload data={results} filename={`${filename}.csv`}>
                <Button
                  leftIcon={<BsFillFileEarmarkArrowDownFill />}
                  colorScheme="blue"
                  fontSize={"xl"}
                >
                  Export CSV
                </Button>
              </CsvDownload>
            </HStack>
          </Stack>
          <OutputTable data={results} />
        </>
      ) : (
        <VStack justifyContent="center" p={4}>
          <Image
            src={girlWithLaptopImage}
            alt="Girl With Laptop"
            maxH="300px"
          />
          <Heading as="h1" fontSize="xl" mt={4}>
            Nothing to show at the moment
          </Heading>
          <Text>Run a query first to see resulting tablet</Text>
        </VStack>
      )}
    </>
  );
};
```

**Review by janvi01 — CHANGES_REQUESTED** [review]
Hey @227001 Thanks for the PR. The code and the implementation look good to me.

One thing here is we can optimize the code, you have implemented separate code for both mobile view and larger view by separate boxes and using `display` attr. This can be done easily using a single `stack` in chakra ui and just setting the direction of the stack as `column` for mobile view and `row` for large screen view.

I have requested the changes, kindly look at that and see where you could optimize. Implement the changes, then this PR would be ready to merge.

**Comment by janvi01** [comment]
@227001 any updates?

**Comment by 227001** [comment]
updated

**Review by janvi01 — APPROVED** [review]
LGTM @227001

**Pull request merged**